src/hooks/useAccommodationsAndOffers.tsx/index.ts

```diff
@@ -36,16 +54,70 @@ export const useAccommodationsAndOffers = (props: SearchTypeProps | void) => {
     }
   );
 
+  const latestQueryParams = data?.latestQueryParams;
+
   const allAccommodations = useMemo(
     () => normalizeAccommodations(data?.accommodations, data?.offers),
     [data]
   );
 
+  // determine from search url if there is a relevant area to focus
+  let focusedEventCoordinates;
+
+  // if search url contains a focusedEvent get the coordinates
```

Comment:
discussed with dave that it is more logical to move this logic to `useCurrentEvents`